Sources/Commands/SwiftBuildTool.swift

```diff
@@ -25,18 +25,22 @@ import enum Utility.ColorWrap
 import protocol Build.Toolchain
 
 import func POSIX.chdir
+import func POSIX.getenv
 
 /// Additional conformance for our Options type.
 extension BuildToolOptions: XcodeprojOptions {}
 
 private enum Mode: Argument, Equatable, CustomStringConvertible {
     case build(Configuration, Toolchain)
     case clean(CleanMode)
+    case repl(Toolchain)
     case usage
     case version
 
     init?(argument: String, pop: () -> String?) throws {
         switch argument {
+        case "repl":
```

Comment:
I think I would prefer to call this `--repl` for now, until we know exactly where we want it to live.